akka-docs/src/main/paradox/general/message-delivery-reliability.md

```diff
@@ -47,7 +47,7 @@ When it comes to describing the semantics of a delivery mechanism, there are
 three basic categories:
 
  * **at-most-once** delivery means that for each message handed to the
-mechanism, that message is delivered zero or one times; in more casual terms
+mechanism, that message is delivered zero or one time; in more casual terms
```

Comment:
Tricky: google suggests "zero or one times" is actually more common than "zero or one time". However perhaps we can find an even better way to rewrite this sentence?

Reply:
> google suggests "zero or one times" is actually more common than "zero or one time".

I didn't notice before. 

But I find [this](https://docs.microsoft.com/en-us/dotnet/standard/base-types/quantifiers-in-regular-expressions) at MS document
> Match zero or one time.

> we can find an even better way to rewrite this sentence?

How about **none or once**?

Reply:
Yeah, I will use this one - `that message is delivered once or not at all`.